packages/quantic/force-app/main/default/lwc/quanticSvg/quanticSvg.js

```diff
@@ -0,0 +1,16 @@
+import { LightningElement, api } from 'lwc';
+// @ts-ignore
+import pageNotFound from './pageNotFound.html';
+// @ts-ignore
+import quanticSvg from './quanticSvg.html';
+
+export default class QuanticSvgLoader extends LightningElement {
```

Comment:
The class name should be the same as the file name. I saw linting rules about that. I'd suggest to rename the class `QuanticSvg`.

Reply:
good catch thanks :)